SIT.Manager/Views/Installation/SelectView.axaml

```diff
@@ -106,16 +103,16 @@
 			</StackPanel>
 
 			<!-- Server -->
-			<StackPanel Grid.Row="1" VerticalAlignment="Center">
+			<StackPanel VerticalAlignment="Center">
 				<TextBlock Text="{DynamicResource SelectViewServerSectionHeading}" TextTrimming="CharacterEllipsis" Classes="FrameHeading"/>
```

Comment:
I'm highlighting it here as well but these textblocks with the headings to the borders can look off at the moment and/or be hard to read.

Reply:
Added shadow effect. You will see it.